### PR TITLE
Thm timings NOT READY FOR MERGE

### PR DESCRIPTION
**Pull request by utunga**
Creating this PR just to represent the changes that we need to figure out how to merge

### CODE REVIEW

File: native_client/deepspeech.cc [hunk 1]
```diff
@@ -537,23 +537,53 @@ ModelState::decode_raw(DecoderState* state)
...
+  float sum = 0.0f;
+  for (size_t i = 0; i < size; i++) {
+    if (logits[i]>0.0f)
+      sum += logits[i] * log2(logits[i]);   
+  }
+  return -sum;
+}
+
 Metadata*
 ModelState::decode_metadata(DecoderState* state)
```
Comment (utunga) [inline native_client/deepspeech.cc]: What's tricky is that the *old* DeepSpeech master branch passed logits into this function now it just passes this 'DecoderState' object that doesnt appear to have the probabilities/logits attached to it... hmmm
